cardano-node/src/Cardano/CLI/Run.hs

```diff
@@ -53,6 +57,10 @@ import           Cardano.Crypto (ProtocolMagicId, RequiresNetworkMagic(..))
 import qualified Cardano.Crypto.Hashing as Crypto
 import qualified Cardano.Crypto.Signing as Crypto
 
+import           Ouroboros.Network.NodeToClient ( AssociateWithIOCP
+                                                , withIOManager
+                                                )
```

Comment:
@Jimbo4350 What does the style guide say about this? Does the stylish-haskell config in this repo do the right thing?

Reply:
I don't think it does however I need to update the style guide re: imports. We want this format if imports exceed 100 chars:

```
import           Ouroboros.Network.NodeToClient 
                       (AssociateWithIOCP,  withIOManager)
```